platform/darwin/src/NSComparisonPredicate+MGLAdditions.mm

```diff
@@ -1,231 +1,12 @@
 #import "NSComparisonPredicate+MGLAdditions.h"
 
+#import "MGLStyleValue_Private.h"
+
 #import "NSPredicate+MGLAdditions.h"
 #import "NSExpression+MGLPrivateAdditions.h"
 
 @implementation NSComparisonPredicate (MGLAdditions)
 
-- (mbgl::style::Filter)mgl_filter {
```

Comment:
My mistake: we only need to keep `FilterEvaluator` around, since that’s responsible for converting `mbgl::Filter`s to NSPredicates as part of the `predicate` property’s getter. `-mgl_filter`, meanwhile, is part of the setter, which is already dead code, so we can remove it.